distrobox-host-exec

```diff
@@ -22,8 +22,9 @@ trap '[ "$?" -ne 0 ] && printf "\nAn error occurred\n"' EXIT
 
 # Defaults
 host_command=""
+install_host_spawn=""
 distrobox_host_exec_default_command="${SHELL:-/bin/sh}"
-host_spawn_version="1.0.2"
+host_spawn_version="1.0.3"
```

Comment:
This should be 1.1.0 seeing here: https://github.com/1player/host-spawn/releases